src/silx/gui/plot/StackView.py

```diff
@@ -544,10 +545,17 @@ def _updateTitle(self):
         self._plot.setGraphTitle(self._titleCallback(frame_idx))
 
     def _defaultTitleCallback(self, index):
-        return "Image z=%g" % self._getImageZ(index)
+        return f"{self._stack_name} z={self._getImageZ(index):g}"
 
     # public API, stack specific methods
-    def setStack(self, stack, perspective=None, reset=True, calibrations=None):
+    def setStack(
+        self,
+        stack,
```

Comment:
We only have get one NXdata signal and name (the default one).

While for 2D image and 2D scatter we get all signals and names.

That's why in this MR I did not add support for auxiliary signals to "volume_3d" data.